go.mod

```diff
@@ -5,17 +5,24 @@ go 1.14
 replace skywalking/network v1.0.0 => ./protocol/gen-codes/skywalking/network
 
 require (
+	github.com/Azure/go-autorest/autorest v0.11.18 // indirect
 	github.com/Shopify/sarama v1.27.2
 	github.com/enriquebris/goconcurrentqueue v0.6.0
 	github.com/golang/protobuf v1.5.2 // indirect
 	github.com/google/go-cmp v0.5.5
+	github.com/gophercloud/gophercloud v0.17.0 // indirect
 	github.com/grandecola/mmap v0.6.0
 	github.com/prometheus/client_golang v1.9.0
+	github.com/prometheus/common v0.15.0
```

Comment:
We need a LICENSE update about Prometheus.

Reply:
Done

Reply:
> Done

You need to update https://github.com/apache/skywalking-satellite/blob/d174963d0a2484e259fab360cc7ef5de9c52f541/dist/LICENSE#L211-L218 , and you'll see "The text of each license is the standard Apache 2.0 license", for standard APL-2.0, you don't need to copy the license file.

> The following components are provided under the Apache License. See project link for details. The text of each license is the standard Apache 2.0 license.

Reply:
Done, thanks.